examples/braket_features/create_custom_circuit_and_custom_local_simulator.ipynb

```diff
@@ -0,0 +1,1296 @@
+{
```

Comment:
<p>This can be fleshed out more; explain what each public method is for, and any conversions that you're doing (in particular the results)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>